src/utils/retry.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @flow
+ * @file A utility function to retry a function which returns a promise
+ * @author Box
+ */
+const DEFAULT_NUM_RETRIES = 2;
+const DEFAULT_INITIAL_DELAY = 500;
+
+const retry = (
```

Comment:
Can we use https://www.npmjs.com/package/promise-retry

Reply:
We also have that super complicated retryNumOfTimes from function.js

Reply:
Also found this https://gitlab.com/snippets/1775781
Bit more readable

Reply:
I ended up using `retryNumOfTimes`. Not the prettiest or most intuitive, but it works